include/whole_body_state_conversions/whole_body_state_interface.h

```diff
@@ -200,77 +227,103 @@ class WholeBodyStateInterface {
     for (const auto &contact_item : contacts) {
       const std::string &contact_name = contact_item.first;
       const ContactState &contact = contact_item.second;
-
       msg.contacts[i].name = contact_name;
-
       // Pose
-      pinocchio::FrameIndex frame_id = pinocchio_model_.getFrameId(msg.contacts[i].name);
-      if (static_cast<int>(frame_id) > pinocchio_model_.nframes) {
+      pinocchio::FrameIndex frame_id = model_.getFrameId(msg.contacts[i].name);
+      if (static_cast<int>(frame_id) > model_.nframes) {
         throw std::runtime_error("Frame '" + contact_name + "' not found.");
       }
-
-      // TODO: Option to retrieve the contact position from an argument (map)
-      const pinocchio::SE3 &oMf = pinocchio::updateFramePlacement(pinocchio_model_, pinocchio_data_, frame_id);
-      pinocchio::SE3::Quaternion oMf_quaternion(oMf.rotation());
-
-      // TODO: Option to retrieve the contact velocity from an argument (map)
-      pinocchio::Motion ovf = pinocchio::getFrameVelocity(pinocchio_model_, pinocchio_data_, frame_id,
-                                                          pinocchio::ReferenceFrame::LOCAL_WORLD_ALIGNED);
-
+      if (!isnan(contact.position.translation().norm())) {
```

Comment:
Norm requires square + sqrt operations. Can't we just check x,y,z individually (isnan(x)||isnan(y)||isnan(z)) - or just one? Same for below with velocity/force/normal